src/IntelliTect.Coalesce/TypeDefinition/MethodViewModel.cs

```diff
@@ -49,9 +50,21 @@ internal MethodViewModel(ClassViewModel parent)
         public string JsVariableModal => JsVariable + "Modal";
         public string JsVariableArgs => JsVariable + "Args";
         public string JsVariableWithArgs => JsVariable + "WithArgs";
+        public string HttpMethodName => HttpMethodType.ToString().ToUpper();
+        public string ApiControllerAnnotation => $"Http{HttpMethodType.ToString()}";
 
 
-
+        public HttpMethodType HttpMethodType
+        {
+            get
+            {
+                if (HasAttribute<HttpMethodAttribute>())
```

Comment:
Tell, don't ask. Wouldn't it be cleaner to ditch the if check altogether and just use the return statement that is inside of the if.

Reply:
Just changed to arrow.